.travis.yml

```diff
@@ -22,7 +22,6 @@ before_install:
   - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
   - gem install xcpretty-travis-formatter --no-rdoc --no-ri --no-document --quiet
   - gem install cocoapods --pre --no-rdoc --no-ri --no-document --quiet
-  - brew install carthage
```

Comment:
This was just in to make explicitly sure we always use the latest version. Same with CocoaPods etc. above. Sometimes the Travis CI infra is not very up-to-date.